### PR TITLE
Update find_duplicate_elements_from_an_array.cpp

### PR DESCRIPTION
**Pull request by AnushkaS09**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Remove Duplicates from a given array

### Description

Yes

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

PR:(#3518)

### CODE REVIEW

File: C-Plus-Plus/other/find_duplicate_elements_from_an_array.cpp [hunk 1]
```diff
@@ -4,30 +4,22 @@ Formally find all duplicate elements of the array. This can be done with the hel
 */
 
 #include <bits/stdc++.h>
+#include<iostream>
+#include<vector>
+#include<unordered_map>
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/other/find_duplicate_elements_from_an_array.cpp]: remove the header

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
add the readme as well

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
Add header file `#include <bits/stdc++.h>`

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
Code produces error cross check missing braces

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
Do the changes mentioned

**Review by ankitaggarwal23 — COMMENTED** [review]
Last Reminder, Are you still working on it.

**Comment by AnushkaS09** [comment]
Yes, I am working on it will do the changes required by today

**Review by ankitaggarwal23 — APPROVED** [review]

**Review by ankitaggarwal23 — COMMENTED** [review]
Problem already present #3518

**Comment by AnushkaS09** [comment]
@ankitaggarwal23 yes but the method is different and shorter I have created this PR on the already existing problem only and have not created an issue of the same problem  again

**Pull request merged**